base/logger/R/logger.R

```diff
@@ -146,7 +146,7 @@ logger.message <- function(level, msg, ..., wrap = TRUE) {
       paste(c(msg, args), collapse = " "),
       "latin-ascii"
     )
-    if (nchar(long.msg) > 20 && wrap) {
+    if (wrap) {
```

Comment:
The hard-coded length in the if control should be removed. The `wrap` argument should fully define it.